tldr.py

```diff
@@ -129,11 +134,15 @@ def get_page_url(command, platform, remote, language):
     return remote + language + "/" + platform + "/" + quote(command) + ".md"
 
 
-def get_page_for_platform(command, platform, remote, language):
+def get_page_for_platform(command, platform, remote, language, only_use_cache=False):
     data_downloaded = False
     if USE_CACHE and have_recent_cache(command, platform, language):
         data = load_page_from_cache(command, platform, language)
     else:
+        if only_use_cache:
```

Comment:
We can combine these to a single statement:

```suggestion
    elif only_use_cache:
```